### PR TITLE
Small Typos in Associations Guide

### PR DESCRIPTION
**Pull request by christopherciufo**
Small typos I found while reading through the docs for the first time

As I am very new to Hanami, can someone confirm that the `where` clause correction makes sense? Thanks.

### CODE REVIEW

File: source/guides/models/associations.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ class AuthorRepository < Hanami::Repository
   end
 
   def find_with_books(id)
-    aggregate(:books).where(authors__id: id).as(Author).one
+    aggregate(:books).where(author_id: id).as(Author).one
```
Comment (jodosha) [inline source/guides/models/associations.md]: This is Sequel syntax, it's correct with double underscore. :wink:

**Comment by jodosha** [comment]
@christopherciufo Hi and thanks for this PR.

> As I am very new to Hanami, can someone confirm that the where clause correction makes sense? Thanks.

Sorry, but I don't understand this question. Can you please tell us an example of what you are trying to achieve? Thanks! :smile:

**Comment by AlfonsoUceda** [comment]
@jodosha maybe is the line you commented.

@christopherciufo thanks for your PR, could you fix what @jodosha said? Thanks!

**Comment by christopherciufo** [comment]
Sorry all for the confusion. This very tiny PR is ready to go haha 👍

**Comment by AlfonsoUceda** [comment]
thanks and merging!

**Pull request merged**